WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardService.swift

```diff
@@ -92,4 +100,20 @@ private extension BlogDashboardService {
 
         return (sections, items)
     }
+
+    func createQuickActionsItems() -> [DashboardCardModel] {
```

Comment:
I don't think this belongs here. I think we can move that to the cell itself (`DashboardQuickActionsCardCell`).

This service/parser doesn't need to know about what items to display. Even parsing the posts here it's not ideal, is something I'm still planning to tackle and remove from here.